lib/helpers/checkr_api_error.py

```diff
@@ -0,0 +1,6 @@
+class CheckerAPIError(RuntimeError):
+    pass
+
+
+class CheckerAPIFailedCreation(CheckerAPIError):
+    pass
```

Comment:
I'd like to know more about how to implement these classes.

Reply:
Usually you don't need to implement them. The error name alone is enough to try-except from. But if you want to you can set a default message with something like this:

```python
class SomeRandomError(RuntimeError):
    def __init__(self, message: Optional[str] = None, **kwargs):
        message = message or "Some default message"
        super().__init__(message, **kwargs)
```

But it is rarely worth the effort, since it is easy enough to just pass the error string in when you raise the error in the first place.

Reply:
Awesome! Thanks for the explanation!